### PR TITLE
Add version 1.12.0 update readme

### PR DESCRIPTION
**Pull request by thelamer**
This adds version 1.12.0 which will be the default version presented to the user.
Also updated the readme to reflect the version change using the latest version of TrueNas scale which has a slightly updated method while setting the settings.

This has been tested on `TrueNAS-SCALE-22.12.0` and is functional with gamepad. I did not fully test GPU compatibility as it is a VM I am using, but Truenas is a Debian distro and all of the devices are mounted in on the privileged container, along with NVIDIA runtime support on the more complex charts at TrueCharts.

### CODE REVIEW

File: stable/kasm/1.12.0/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Introduction
+
+Kasm Workspaces streaming containerized apps and desktops to end-users.
+
+TrueCharts are designed to be installed as TrueNAS SCALE app only. We can not guarantee this charts works as a stand-alone helm installation.
+**This chart is not maintained by the upstream project and any issues with the chart should be raised [here](https://github.com/truecharts/apps/issues/new/choose)**
+
+## Source Code
+
+* <https://github.com/thelamer/charts-test/tree/main>
```
Comment (rickkoliser) [inline stable/kasm/1.12.0/README.md]: do we want to be pointing at your repo, or the kasmtech repo?

Comment (thelamer) [reply stable/kasm/1.12.0/README.md]: No we sure don't, updated both of them.

File: stable/kasm/1.12.0/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Installing the Chart
+
+To install this App on TrueNAS SCALE check our [Quick-Start Guide](https://truecharts.org/manual/Quick-Start%20Guides/03-Installing-an-App/).
+
+## Uninstalling the Chart
+
+To remove this App from TrueNAS SCALE check our [Quick-Start Guide](https://truecharts.org/manual/Quick-Start%20Guides/07-Deleting-an-App/).
+
+## Support
```
Comment (rickkoliser) [inline stable/kasm/1.12.0/README.md]: forgive my ignorance on what is common for Truenas projects (or anything you and Justin discussed on this), but all these support options seem to point to truenas specific support, should any of these point to our public issue tracker on github?

Comment (thelamer) [reply stable/kasm/1.12.0/README.md]: This readme is not presented to the user it is a git thing and was copied over from truecharts for the most part including the library for compatibility.  Though the app-readme is, I have pulled out those references as they are not relevant outside of using their library.

**Comment by rickkoliser** [comment]
Everything looks good, approved.

**Pull request merged**